src/pages/Attendance.jsx

```diff
@@ -1,11 +1,220 @@
+import React from "react";
+import dayjs from "dayjs";
+import "dayjs/locale/ko";
+import { Stack, styled } from "@mui/material";
+import Chip from "@mui/material/Chip";
+import Typography from "@mui/material/Typography";
+import Checkbox from "@mui/material/Checkbox";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import ChurchIcon from "@mui/icons-material/Church";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import SettingsIcon from "@mui/icons-material/Settings";
 import Layout from "../components/Layout/Layout";
+import Title from "../components/shared/Title";
+import mcyMember from "../components/shared/member.js";
 
 const Attendance = () => {
+  const [value, setValue] = React.useState(dayjs());
+  const [isOpenCalendar, setIsOpenCalendar] = React.useState(false);
+  const [selectedLeader, setSelectedLeader] = React.useState(null);
+  const [isChecked, setIsChecked] = React.useState(false);
+
+  const handleChange = (newData) => {
+    setValue(newData);
+    setIsOpenCalendar(false);
+  };
+
+  const handlePrevDayClick = () => {
+    const newDate = dayjs(value).subtract(1, "day");
+    setValue(newDate);
+  };
+
+  const handleNextDayClick = () => {
+    const newDate = dayjs(value).add(1, "day");
+    setValue(newDate);
+  };
+
+  const handleCalendarClick = () => {
+    setIsOpenCalendar(true);
+  };
+
+  const handleLeaderClick = (cell) => {
+    setSelectedLeader(cell);
+  };
+
+  const handleCheck = () => {
+    setIsChecked(true);
+  };
   return (
     <Layout>
-      <div>Attendance</div>
+      <AttendanceWrapper>
+        <TitleWrapper sx={{ height: "20%" }}>
```

Comment:
```suggestion
        <TitleWrapper sx={{ height: "20%" }}>
```
이렇게 styled component로 만든 부분에서는 sx를 쓸 필요가 없음.
실제 스타일드컴포넌트에
height:20% 를 적용하도록. 다른 부분도 다 변경하길